### ISSUE TITLE
Certus Quartz config seems to ignore values

### ISSUE THREAD

**Issue by Raidobw2** [AppliedEnergistics/Applied-Energistics-2#6272]
### Describe the bug

In the `ae2-common.toml` config:
```
[worldGen]
	spawnPressesInMeteorites = true
	generateQuartzOre = true
	quartzOresPerCluster = 1
	quartzOresClusterAmount = 1
	quartzOresBiomeBlacklist = []
```
If you set the values to 1 and 1 for certus quartz which should make 1 cluster of 1 ore block per chunk, you instead still see the ratio of 7 ores per cluster, 20 clusters per chunk.

### How to reproduce the bug

1. Change the values to 1 and 1
2. Start game and go in spectator check oregen
3. Notice Certus Quartz is in big amounts despite the config changes

### Expected behavior

Changing the config should work

### Additional details

Here's what a 1 and 1 config looked like for me:
![image](https://user-images.githubusercontent.com/17010632/168503466-c2cb0b82-5a5a-492d-bf17-078a1dd53a95.png)

I'll have a snippet of a manual crash below with system info and loaded mods (hopefully that'll look ok).

Thank you in advance

### Which minecraft version are you using?

1.18

### On which mod loaders does it happen?

Forge

### Crash log

```
-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Mod Resources, ResourcefulCreepers

-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1554663168 bytes (1482 MiB) / 2998927360 bytes (2860 MiB) up to 8522825728 bytes (8128 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 10
	Microarchitecture: Coffee Lake
	Frequency (GHz): 3.70
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1080
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1b80
	Graphics card #0 versionInfo: DriverVersion=30.0.14.9649
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 8192.00
	Memory slot #2 clockSpeed (GHz): 2.67
	Memory slot #2 type: DDR4
	Memory slot #3 capacity (MB): 8192.00
	Memory slot #3 clockSpeed (GHz): 2.67
	Memory slot #3 type: DDR4
	Virtual memory max (MB): 37569.73
	Virtual memory used (MB): 17150.41
	Swap memory total (MB): 4864.00
	Swap memory used (MB): 29.08
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8128m -Xms256m
	Launched Version: forge-40.1.19
	Backend library: LWJGL version 3.2.2 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1080/PCIe/SSE2 GL version 3.2.0 NVIDIA 496.49, NVIDIA Corporation
	Window size: 1360x705
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'; Server brand changed to 'forge'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fast
	Resource Packs: vanilla, mod_resources, file/ResourcefulCreepers
	Current Language: English (US)
	CPU: 12x Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
	Server Running: true
	Player Count: 1 / 8; [ServerPlayer['NMX_R3GEN'/68, l='ServerLevel[New World]', x=3839.64, y=-1.74, z=9959.99]]
	Data Packs: vanilla, mod:yungsapi, mod:theoneprobe (incompatible), mod:roughlyenoughitems (incompatible), mod:cloth_config (incompatible), mod:forge (incompatible), mod:paxi, mod:architectury (incompatible), mod:rei_internals_workaround (incompatible), mod:ae2 (incompatible)
	World Generation: Stable
	ModLauncher: 9.1.3+9.1.3+main.9b69c82a
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		 mixin PLUGINSERVICE 
		 eventbus PLUGINSERVICE 
		 slf4jfixer PLUGINSERVICE 
		 object_holder_definalize PLUGINSERVICE 
		 runtime_enum_extender PLUGINSERVICE 
		 capability_token_subclass PLUGINSERVICE 
		 accesstransformer PLUGINSERVICE 
		 runtimedistcleaner PLUGINSERVICE 
		 mixin TRANSFORMATIONSERVICE 
		 fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		javafml@null
	Mod List: 
		client-1.18.2-20220404.173914-srg.jar             |Minecraft                     |minecraft                     |1.18.2              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		YungsApi-1.18.2-Forge-2.0.8.jar                   |YUNG's API                    |yungsapi                      |1.18.2-Forge-2.0.8  |DONE      |Manifest: NOSIGNATURE
		theoneprobe-1.18-5.1.0.jar                        |The One Probe                 |theoneprobe                   |1.18-5.1.0          |DONE      |Manifest: NOSIGNATURE
		RoughlyEnoughItems-8.1.457.jar                    |Roughly Enough Items (REI)    |roughlyenoughitems            |8.1.457             |DONE      |Manifest: NOSIGNATURE
		cloth-config-6.2.62-forge.jar                     |Cloth Config v4 API           |cloth_config                  |6.2.62              |DONE      |Manifest: NOSIGNATURE
		forge-1.18.2-40.1.19-universal.jar                |Forge                         |forge                         |40.1.19             |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		Paxi-1.18.2-Forge-2.0.jar                         |Paxi                          |paxi                          |1.18.2-Forge-2.0    |DONE      |Manifest: NOSIGNATURE
		architectury-4.4.60.jar                           |Architectury                  |architectury                  |4.4.60              |DONE      |Manifest: NOSIGNATURE
		RoughlyEnoughItems-jei-internals-workaround-8.1.45|REI Internals Workaround      |rei_internals_workaround      |8.1.457             |DONE      |Manifest: NOSIGNATURE
		appliedenergistics2-11.0.0.jar                    |Applied Energistics 2         |ae2                           |11.0.0              |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: f7a61d2d-3222-4f21-83cc-0fc69a88a3e5
	FML: 40.1
	Forge: net.minecraftforge:40.1.19
```

**Comment by Raidobw2** [comment]
Hello, sorry I forgot to specify! Yes that was a brand new world. You had me in doubt, so I doublechecked the config, launched the game and made another new one. Here's pictures of this other new world:
![image](https://user-images.githubusercontent.com/17010632/168546526-8785e99d-5c51-4081-adda-258717cc1d6f.png)
![image](https://user-images.githubusercontent.com/17010632/168546679-06cc56f4-c075-4890-a7e3-2fb95808c140.png)

**Comment by Technici4n** [comment]
What's actually happening is that Forge, in their infinite wisdom, decided that configs should be loaded after worldgen features are registered.

**Comment by Technici4n** [comment]
Should be fixed in 11.1.0 (soon on CurseForge).

**Comment by Raidobw2** [comment]
Thank you and have a nice day!
